engine/src/resource/json_utils.cpp

```diff
@@ -11,7 +108,7 @@
  * Vega Strike is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
+ * any later version.
```

Comment:
Please don't touch the legal text in one file only. Changes should happen project wide and after approval by key stakeholders. It's very problematic to change the legal license of existing files, as they belong to those who worked on them.

I say this as someone who is part of the second dev team from 2019. This project is 25 years old and the original developers still hold rights here, including the choice they made to release this in GPL v.3.